ITEventsTests/SearchParametersPresenterTests/SearchParametersPresenterTests.swift

```diff
@@ -0,0 +1,120 @@
+import XCTest
+@testable import ITEvents
+
+class SearchParametersPresenterTests: XCTestCase {
+    var searchParametersPresenter: ISearchParametersPresenter!
+    var view: (ISearchParametersView & ISearchParametersViewMock)!
+    var searchParametersService: ISearchParametersService!
+    
+    override func setUp() {
+        super.setUp()
+        
+        searchParametersService = SearchParametersService()
+        view = SearchParametersMockViewController()
+        searchParametersPresenter = SearchParametersPresenter(view: view, searchParametersService: searchParametersService)
+    }
+    
+    override func tearDown() {
+        searchParametersPresenter = nil
+        view = nil
+        searchParametersService = nil
+        super.tearDown()
+    }
+    
+    func testPresenterSetup() {
+        //Given
+        let selectedTags = [Tag(id: 1, name: "JavaScript"),
+                            Tag(id: 2, name: "iOS")]
+        let otherTags = [Tag(id: 3, name: "Android"),
+                         Tag(id: 4, name: "Python")]
+        searchParametersService.updateTags(selectedTags: selectedTags, otherTags: otherTags)
+        
+        //When
+        searchParametersPresenter.setup()
+        
+        //Then
+        XCTAssertEqual(view.selectedTags, searchParametersService.selectedTags)
+        XCTAssertEqual(view.otherTags, searchParametersService.otherTags)
+    }
+    
+    func testPresenterSelectSearchTagAndSave() {
+        //Given
+        let selectedTags = [Tag(id: 1, name: "JavaScript"),
+                            Tag(id: 2, name: "iOS")]
+        let otherTags = [Tag(id: 3, name: "Android"),
+                         Tag(id: 4, name: "Python")]
+        let finalSelectedTags = [Tag(id: 3, name: "Android"),
```

Comment:
Good to prefix all expectations with `expected` prefix.
So may be both variables should be renamed to:
```swift
let expectedSelectedTags = 
let expectedOtherTags =
```

and may be moved under ```//Then``` section? (not sure)